subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testng/TestNGTestFrameworkIntegrationTest.groovy

```diff
@@ -74,4 +75,19 @@ class TestNGTestFrameworkIntegrationTest extends AbstractTestFrameworkIntegratio
     String getFailingTestCaseName() {
         return "fail"
     }
+
+    @Issue("GRADLE-3545")
```

Comment:
Could you use the GitHub issue URL here and in the other places e.g. `Issue("https://github.com/gradle/gradle/issues/3545")`.

Reply:
In retrospect, this makes a lot of sense. Updated!